cpp/src/components/connectivity.cu

```diff
@@ -205,8 +201,8 @@ void connected_components(Graph *graph,
 {
   cudaStream_t stream{nullptr};
 
-  CUGRAPH_EXPECTS(table != nullptr, "Invalid API parameter");
-  CUGRAPH_EXPECTS(table->num_columns() > 1, "Invalid API parameter");
+  CUGRAPH_EXPECTS(table != nullptr, "Invalid API parameter: table parameter is NULL");
+  CUGRAPH_EXPECTS(table->num_columns() > 1, "Invalid API parameter: table has no columns");
```

Comment:
I think error message here is misleading. If num_columns() == 1 (so table has "one" column), this assertion still fails.

Reply:
yep.  the test really should have been "> 0"

fixed